assets/js/components/notifications/ZeroDataStateNotifications/index.js

```diff
@@ -49,12 +49,16 @@ export default function ZeroDataStateNotifications() {
 		? MODULES_ANALYTICS_4
 		: MODULES_ANALYTICS;
 
+	const analyticsModuleName = isGA4DashboardView
+		? 'analytics-4'
+		: 'analytics';
```

Comment:
Since this is only used in the `useSelect` below, how about we move it within that function?

Reply:
Actually, we have the same expression repeated above in the `isAnalyticsConnected` `useSelect()` call - this constant could be moved up, and reused...